docs/meshcloud.index.md

```diff
@@ -25,6 +25,11 @@ The figure and table below explain the relation of the most important concepts i
 | [meshPlatform](meshcloud.platform-location.md) | An individual cloud platform connected to meshcloud.                                      |
 | [meshLocation](meshcloud.platform-location.md) | A grouping of meshPlatforms, e.g. by geographic region.                                   |
 
+
+## meshPanel supported browser
+Firefox & Chrome
```

Comment:
Our browserlist file currently has 

```
Firefox ESR
last 2 Firefox versions
last 2 Chrome versions
```

Maybe we should clarify that? We also have other browsers supported on a best effort basis (mainly Edge(Chromium), Safari + mobile), which means any bugs don't count as defects per our terms. We're still happy to learn about them though.

Reply:
@chewbeccy ping :-)